src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -710,6 +710,29 @@ protected bool IsAlive
                 return !ElementIDLifecycleManager<int>.GetInstance().IsRevitDeleted(InternalElementId.IntegerValue);
             }
         }
+        /// <summary>
+        /// Finds the elements that are joined with the given element.
+        /// </summary>
+        /// <returns>all elements joined to the given element</returns>
+        public List<Element> GetJoinedElements()
+        {
+            if (this.InternalElement == null)
+                throw new NullReferenceException("Cannot get joined elements from a null Element");
+            if (Document == null)
+                throw new NullReferenceException("Cannot get joined elements from a null Element");
+
+            ElementId[] joinedElementIds = JoinGeometryUtils.GetJoinedElements(Document, this.InternalElement)
+                                                            .ToArray();
+            
+            List<Element> joinedElements = new List<Element>();
```

Comment:
since we're getting an `ElementId[]` array above, we should also use an array here and pre-allocate the size of it. This is always faster than `List<>` elements and in this case, the additional performance matters.